src/Compilers/Core/Portable/Binding/BindingDiagnosticBag.cs

```diff
@@ -76,7 +76,7 @@ protected BindingDiagnosticBag(bool usePool)
 
         internal bool AccumulatesDependencies => DependenciesBag is object;
 
-        internal void Free()
+        internal virtual void Free()
```

Comment:
Is this necessary?

Reply:
Ah, never mind. I see the override now. I was looking at the wrong diff window. Thanks